utils/luci-mod-freifunk-ui/Makefile

```diff
@@ -10,7 +10,7 @@ include $(TOPDIR)/rules.mk
 LUCI_TITLE:=Freifunk Public and Admin LuCI UI
 LUCI_DEPENDS:=+luci-mod-admin-full +luci-lib-json +community-profiles +luci-lib-ipkg
 LUCI_PKGARCH:=all
-PKG_RELEASE:=3
```

Comment:
No need to bump this, as it's only a small code-change (which will create a new version-number from the git-commit-header by the Makefile-macros) and no change in the package-structure or functionality happens.